core/basic_models/actions/basic_actions.py

```diff
@@ -118,16 +118,16 @@ def run(self, user: BaseUser, text_preprocessing_result: BaseTextPreprocessingRe
 
 class ChoiceAction(Action):
     version: Optional[int]
-    requirement_actions: RequirementAction
-    else_action: Action
+    items: List[RequirementAction]
```

Comment:
```suggestion
    items: List[RequirementAction]
```
в __init__: self.items = self.build_items(), где build_items(self) -> List[str]

Reply:
верно, `build_items` возвращает список строк, но над функцией есть декоратор `@list_factory(RequirementAction)`, из-за которого сами `items` уже являются списком `RequirementAction`